src/Quartz.HttpClient/HttpApiContract/Dtos.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Quartz.HttpApiContract;
+
+internal record KeyDto(string Name, string Group)
+{
+    public static KeyDto Create(JobKey jobKey) => new(jobKey.Name, jobKey.Group);
+
+    public static KeyDto Create(TriggerKey triggerKey) => new(triggerKey.Name, triggerKey.Group);
+
+    public JobKey AsJobKey() => new(Name, Group);
+
+    public TriggerKey AsTriggerKey() => new(Name, Group);
+
+    public override string ToString() => Group + '.' + Name;
+}
+
+internal record NamesDto(IReadOnlyCollection<string> Names);
+
+internal record SchedulerContextDto(Dictionary<string, string> Context)
+{
+    public static SchedulerContextDto Create(SchedulerContext context)
```

Comment:
I like this `Create` pattern!

Reply:
I copied it from the preliminary HTTP API work done by you 😄